### ISSUE TITLE
[Bug] Problema with window size when upgrading to 1.7.x - not an issue in 1.6.4

### ISSUE THREAD

**Issue by fbaldner** [OpenBoard-org/OpenBoard#887]
### Describe the bug
When installing the new version 1.7.0 on my desktop, I get a limited window to draw one, that seems like a "slit" from the whole page. Like this one:

![OpenBoard_M6ahCIdicD](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/46d5bf84-fc8e-4da8-8806-5ce885cd2ac4)

The gray area can't be drawn into.
If I go in the Documents part, the page preview is shown correctly (the page in question is in the third row, second column):

![OpenBoard_vlyvh4CNad](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/49ab7863-0cbf-416a-a783-93d18a2b977f)

This also happens with version 1.7.1-rc240202

If I install version 1.6.4 the page is shown correctly:

![OpenBoard_7h7cbZG5jI](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/cfbb80c1-1721-4290-8e1b-2972a5db5f6f)

Even if I now go back and install 1.7.x it will go back to that "slit" version of the page.

On my laptop, I have a weird zoom on my board:

![Openboard na v1 7 0](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/ec7b7086-692d-4c0c-9a8c-fcf012a4b3e6)

In this version, on my laptop, if I zoom out, it only affects that page, not the whole document.

Whereas if I install 1.6.4 I get a regular view:

![Openboard na v1 6 4](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/905f432c-d5b3-44bf-bc65-562447d4fa25)

### To Reproduce

1. Install version 1.7.0/1.7.1-rc240202
2. Open it and realizes either the "slit" (desktop) or the "weird zoom" (laptop)
3. Close OpenBoard
4. Install version 1.6.4
5. Open it and realizes that both are ok.

#### Expected behavior
When opening the board, I should have a proper view that allows me to write into my whole board, not just a specific area of the page or with the tools on top of the board.

#### Actual behavior
I get a cut view of the board in my desktop and a zoomed version in my laptop.

### Screenshots
Screenshots are above, in the problem description.

### Additional resources
*PDF, images, sounds... anything useful to reproduce the bug*

### Context

  - reproducibility : `systematic` `not systematic`
  - OpenBoard version :
      - bug happens with : `1.7.0` `1.7.1rc-240202z`
      - bug doesn't happen with : `1.6.4`
      - didn't check : previous version to `1.6.4`
  - OS : 
      - bug happens with : `Windows 10`
      - bug doesn't happen with : didn't test in other OS
      - didn't check : `Linux` `macOS` 
  - Installation source : `OpenBoard GitHub download page`
  - If you built OpenBoard from source, please provide the branch you're working on : `master`
  
### Additional context

- OS(es) detailed version (for the ones you reproduced the bug on) 
    - Windows : `desktop: Microsoft Windows 10 Pro Version 10.0.19045 Build 19045` `laptop: Microsoft Windows 10 Home Single Language Version 10.0.19045 Build 19045`
- number and resolution of monitors : `desktop: 2 monitors, one 2560x1080 (OpenBoard is used on this one) and the other 1080x2560` `laptop: 1 monitor, 1366x768`
- if used, manufacturer and model of graphic tablet input device : ` Wacon One, but this problem happens with or without it`

**Comment by letsfindaway** [comment]
Thanks for reporting. I think you spotted a problem not so obvious to others not using such an "extreme" monitor configuration. Is it correct that you have both monitors enabled, the landscape and the portrait one? That is an unusual configuration, but nevertheless OpenBoard should cope with it.

The idea in OpenBoard was to indicate to the user on the control screen (the one with the controls) what is visible on the display screen (the one only showing the board). The area not visible on the display screen should also be hidden on the control screen. That makes sense and avoids that the teacher operates on a part of the board which is invisible to the students.

But even on my "not so extreme" monitor configuration I can see an incorrect behavior. So it seems that the boxing of the visible area is not correctly implemented.

Verified this problem on Linux, so it is probably not OS dependent.

Can you verify that the display on the control screen is correct when you switch to single monitor in the OpenBoard menu?

**Comment by fbaldner** [comment]
Yup! I do have both monitors enabled in this configuration, The main one, landscape, and the "reading" one, portrait.

When you say control screen and display screen, you're refering to the multi screen option being enabled? I don't use that on, but, if that is, I can check for you. If not, please guide me through.

**Comment by letsfindaway** [comment]
> Yup! I do have both monitors enabled in this configuration, The main one, landscape, and the "reading" one, portrait.
> 
> When you say control screen and display screen, you're refering to the multi screen option being enabled? I don't use that on, but, if that is, I can check for you. If not, please guide me through.

Yes, you can call it "main" and "reading", I call it "control" and "diisplay". You can disable multi-monitor mode here:
![grafik](https://github.com/OpenBoard-org/OpenBoard/assets/35591413/13739caa-4062-4214-b808-26a10972741e)
It is the option which is selected on the screenshot. In German, it is called "Mehrere Biuldschirme".

**Comment by fbaldner** [comment]
This is for 1.6.4:

![OpenBoard v1 6 4 on Multi Screen](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/2c3e0893-8ec0-4586-ad27-1c56a80166ae)

Which made me understand now what was all about the problem with 1.7.0, since it's trying to "mimick" the shape of the display monitor.

Now for 1.7.0, one thing I noticed is that, if I load OpenBoard, get into Multi Screen mode , load the document, it'll show up like this:

![OpenBoard v1 7 0 on Multi Screen](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/b7dd0aa2-91ad-4222-94f9-fb532d3c024e)

So yeah, display is okay, but control is still in the weird zoom thing.

If I exit Multi Screen mode and get back to it, this is the result:

![OpenBoard v1 7 0 on Multi Screen (2)](https://github.com/OpenBoard-org/OpenBoard/assets/35616989/bb77badf-ea41-4f43-9a45-be325e7e56e8)

I can even load another document that the display monitor will still be blank. I tried loading another and then the original document, nothing, all blank. If I exit the program and re-do what I described in the first picture for 1.7.0, it'll show up correctly in the display monitor, even though the control monitor isn't proper.

As for my laptop, I can't test now with a second monitor, but later tonight I can get a couple minutes earlier in class and test it out with the projector. To be honest, I've never tried this multi screen option. But I think I'll start using it now, since the students will have a much cleaner view of the document, without all the tools.

A question now: is there any way to output this display mode to a new window? My point here is to try and send that window to OBS or even share a window in Zoom/Google Meet, in case I want to present it without the clutter of the tools, just the document and what I'm scribbling.

**Comment by elbecerrasoto** [comment]
Same, problem on `Manjaro Linux x86_64 `.
My display configuration is as shown on the image:

### Display configuration
![20240327_062741](https://github.com/OpenBoard-org/OpenBoard/assets/16927293/9f619247-0aac-4923-b8ff-1f417fe4bace)


### Screenshot for `1.7.0`:
`flatpak install ch.openboard.OpenBoard`

![20240327_063015](https://github.com/OpenBoard-org/OpenBoard/assets/16927293/4c3b06af-6aab-4193-acf8-744a3dc20986)

### Screenshot for `1.6.4`:
`flatpak update --commit=1fb3826ebe2b40271c8f16cf184ef4c9acc58a80efe0e7d46b7a4a13dd621561 ch.openboard.OpenBoard`

![20240327_065143](https://github.com/OpenBoard-org/OpenBoard/assets/16927293/ded0d6b9-559f-4b98-bbd3-a64f2b36ea1f)


### System Info
```
ebecerra@muss 
------------- 
OS: Manjaro Linux x86_64 
Host: Latitude 5430 
Kernel: 6.7.7-1-MANJARO 
Uptime: 1 hour, 13 mins 
Packages: 1950 (pacman), 17 (flatpak) 
Shell: zsh 5.9 
Resolution: 1920x1080 
DE: Plasma 5.27.11 
WM: kwin 
WM Theme: ActiveAccentDusk 
Theme: [Plasma], Breeze [GTK3] 
Icons: [Plasma], breeze [GTK2/3] 
Terminal: konsole 
Terminal Font: IBM Plex Mono 18 
CPU: 12th Gen Intel i7-1265U (12) @ 4.800GHz 
GPU: Intel Alder Lake-UP3 GT2 [Iris Xe Graphics] 
Memory: 10858MiB / 31786MiB 
```
